operator/controllers/seldondeployment_controller.go

```diff
@@ -547,7 +547,7 @@ func (r *SeldonDeploymentReconciler) createComponents(ctx context.Context, mlDep
 				pu := machinelearningv1.GetEnginePredictiveUnit(&p.Graph)
 				if pu == nil {
 					// below should never happen - if it did would suggest problem in webhook
-					return nil, fmt.Errorf("Engine not separate and no pu with localhost service - not clear where to inject engine")
+					return nil, fmt.Errorf("engine not separate and no pu with localhost service - not clear where to inject engine")
```

Comment:
Is this required?

Reply:
Error messages shouldn't start with capital letters since they can get concatenated, it's a Golang style thing. Obviously not essential but following the boy scout rule of improving things when you see them